pkg/slo-controller/noderesource/plugins/resourceamplification/plugin.go

```
@@ -111,3 +125,50 @@ func (p *Plugin) Calculate(_ *configuration.ColocationStrategy, node *corev1.Nod
 		},
 	}, nil
 }
+
+func calculateCPUAmplificationRatio(node *corev1.Node, ratio extension.Ratio) extension.Ratio {
+	var finalRatio extension.Ratio = 1
+	if ratio > 0 {
+		finalRatio = ratio
+	}
+
+	// do not return error, print warning log and use config amplification ratio
```

I think we still need to return the error here.